metrics/coverage_high_water_mark

```diff
@@ -1 +1 @@
-93.7300
\ No newline at end of file
+93.6800
```

Comment:
Asking for two-developer forgiveness on this one.  Not super concerned about unit testing logging.